src/components/GroceryListItem.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react'
+import { StyleSheet, Dimensions } from 'react-native'
+import { RectButton } from 'react-native-gesture-handler'
+import CheckBoxButton from '../components/CheckBoxButton';
+import IconButton from '../components/IconButton';
+import { Box, Text } from '../components/index';
+import Svg, { G, Path, Rect, TSpan } from "react-native-svg";
+
+const windowWidth = Dimensions.get('window').width;
+
+const styles = StyleSheet.create({
+    groceryList: {
+        marginTop: 17,
+        alignSelf: "flex-start",
+        marginLeft: 15 //used to be 39
+     },
+    item: {
+        marginLeft: 16
+    },
+    container: {
+        width: windowWidth - 64
+    }
+});
+
+const HamburgerIcon = () => {
+    return (
+        <Svg width={21.546} height={15} viewBox="0 0 21.546 15">
+      <G fill="none" stroke="#d0ced5" strokeLinecap="round" strokeWidth={3}>
+        <Path data-name="Path 52" d="M1.5 1.5h18.546" />
+        <Path data-name="Path 53" d="M1.5 7.5h18.546" />
+        <Path data-name="Path 54" d="M1.5 13.5h18.546" />
+      </G>
+    </Svg>
+    )
+}
+
+const listItems = ["Hot Dogs", "Bananas", "Red Onions", "Bacon"];
```

Comment:
Please make sure that Rajeev/myself is aware of how this `listItems` variable should be populated. That way, we don't have hardcoded values forever.

Reply:
I'm not exactly sure how listItems is going to be populated in the future. It may or may not also include the item quantity. I'll keep you updated though.

Reply:
@BlaineSieck so all items are going to be in the form of a FoodItem. I also think the FoodItem should be passes as a prop to the GroceryListItem. I not a React wizard or anything but just a thought... @ynigoreyes  on this as well?